cumulusci/cli/tests/test_cci.py

```diff
@@ -396,23 +396,24 @@ def test_cli():
     "cumulusci.cli.cci.get_latest_final_version",
     mock.Mock(return_value=pkg_resources.parse_version("100")),
 )
-@mock.patch("click.echo")
-def test_version(echo):
+def test_version(capsys):
     run_click_command(cci.version)
-    assert cumulusci.__version__ in echo.call_args_list[1][0][0]
+    console_output = capsys.readouterr().out
+    assert cumulusci.__version__ in console_output
```

Comment:
It might be good to make this a bit more precise, such as `assert f"CumulusCI version:  {cumulusci.__version__)" in  console_output`. While it seems highly unlikely, if someone did a refactor late at night they could accidentally swap version numbers between python and cci and this test wouldn't catch it.

Reply:
Ya this could definitely be made better. I always wonder if there is a "best practice" for assertions on lots of different string that could potentially be displayed to a user during program execution.